src/commands/push-to-docs-gcs.yml

```diff
@@ -5,11 +5,12 @@ parameters:
   gcp-service-key:
     description: |
       The credentials for the Service Account to be used when communicating to Google Cloud.
-      If not provided, the command will look for the environment variable $DOCS_PASSWORD
+      If not provided, the command will look for the environment variable $DOCS_PASSWORD or $GCLOUD_SERVICE_KEY
 
       To obtain the key for this command,
       use either the Google Cloud Platform Console or gcloud iam service-accounts keys create.
       The key can be .json (preferred) or .p12 (legacy) format.
+      Add the new key as a secret in a context in CircleCI.
```

Comment:
Is it really necessary? 
Anyways; I believe this sentence would better suit on line 8-9

Reply:
I'm not really sure about that whole paragraph tbh. I understand it is meant as a guideline for non entur users, but the fact that we reference environment variables in the paragraph above should be clue enough...?
 
Which option should we take:

1. Remove the whole paragraph?
2. Remove the new sentence?
3. Rewrite the paragraph?